content/en/community/contribution-guidelines.md

```diff
@@ -0,0 +1,233 @@
+[Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco).
+By submitting pull requests submitters acknowledge they grant the
+[Apache License v2](./LICENSE) to the code and that they are eligible to grant this license for all commits submitted in their pull requests.
+
+## How to contribute
+
+CDEvents welcomes all of contributions! We encourage you to engage with the
+[CDEvents community](https://github.com/cdevents/spec/blob/main/README.md#community).
+
+Resources for contributors:
+
+* [Project Roadmap](/community/roadmap)
+* [Specification v0.2 Roadmap](https://github.com/orgs/cdevents/projects/1/views/9)
+* [GitHub Issues](https://github.com/cdevents/spec/issues)
+* [Communication Channels](https://github.com/cdevents/spec/blob/main/README.md#community)
+
+## Overview
+
+Documentation is published at the [CDEvents website](https://cdevents.dev) and the source for this site may be found in this [GitHub repo](https://github.com/cdevents/cdevents.dev).
+
+The specification itself is maintained in this [GitHub repo](https://github.com/cdevents/spec).
+
+## Getting set up
+
+To contribute to the documentation, start by forking this [GitHub repo](https://github.com/cdevents/cdevents.dev)
+
+Clone a local copy of your fork with:
+
+```bash
+git clone https://github.com/your-fork/cdevents.dev
+```
+
+This gets you a local copy of your fork of the website as a dev environment.
+
+If you want to be able to contribute to the spec itself, fork the specification [GitHub repo](https://github.com/cdevents/spec)
+
+Get a local copy, in another location, with:
+
+```bash
+git clone https://github.com/your-fork/spec`
+```
+
+Make edits to the website or specification in your fork and contribute them as PRs to the upstream repo. Please squash commits using `git rebase -i` before creating a PR.
+
+Maintainers, please remember to merge PRs for the spec repo, tag them, and then update the cdevents.dev repo to point to the new release of the spec, using the process [here](#versioning-and-releases)
+
+Note that the version of the spec linked from the website is automatically synchronized using the `params.version` variable set in `config.toml`, so this will be updated when you revise the website version.
+
+## Viewing your edits
+
+Install a copy of [Hugo](https://gohugo.io) in your dev environment. You will need a version >= that specified in HUGO_VERSION in [the config](https://github.com/cdevents/cdevents.dev/blob/main/netlify.toml)
+
+From within your local copy of your cdevents.dev fork, start the Hugo server with the following:
+
+```bash
+ hugo server --printI18nWarnings --bind 0.0.0.0 --baseURL http://your-dev-ip-address:1313
+ ```
+<!-- markdownlint-disable-next-line MD001 MD034 -->
+ This will allow you to browse your copy of the website at http://localhost:1313 and also use a phone or tablet to check mobile rendering by navigating to http://your-dev-ip-address:1313 from the device.
```

Comment:
I still would prefer `netlify dev` here. 
Using `hugo` directly bypasses any setting that we add to `netlify.toml`.

Reply:
The problem is, that you typically need to change your hugo settings quite regularly in order to debug what is going on with rendering problems, work around other services running on ports, etc. I've used different flag combos about half a dozen times in the past 24 hours.

We should probably document both alternatives.

Reply:
Instructions updated.